packages/test/test-service-load/README.md

```diff
@@ -4,18 +4,6 @@ _Note: This tool has dependencies on Microsoft-internal systems._
 
 NodeJs-based test to simulate many clients and a high rate of op generation.
 
-<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_README_HEADER) -->
```

Comment:
Note for reviewers: this package isn't a library, it just contains tests. So the library header really isn't appropriate.